debian/changelog

```diff
@@ -1,3 +1,10 @@
+system76-driver (20.04.36~~alpha) focal; urgency=low
+
+  * Thelio-mira-b1: Added quirk for Nvidia GPUs for drivers 460 and 465
```

Comment:
The DAILY WIP message from bump-version has to be left in, for legacy reasons.

Reply:
See commits below, lots of minor adds,removes,and tweaks.